ui/src/components/Navigation/Navigation.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { renderComponent } from "test/utils";
+import { screen, within } from "@testing-library/dom";
+
+import { urls } from "urls";
+
+import Navigation from "./Navigation";
+import { Label } from "./types";
+import { authURLs } from "api/auth";
+import { appendAPIBasePath } from "util/basePaths";
+
+test("displays page links", () => {
+  renderComponent(<Navigation />);
+  expect(
+    screen.getByRole("link", { name: Label.IDENTITY_PROVIDERS }),
+  ).toHaveAttribute("href", urls.providers.index);
+  expect(screen.getByRole("link", { name: Label.CLIENTS })).toHaveAttribute(
+    "href",
+    urls.clients.index,
+  );
+  expect(screen.getByRole("link", { name: Label.IDENTITIES })).toHaveAttribute(
+    "href",
+    urls.identities.index,
+  );
+  expect(screen.getByRole("link", { name: Label.SCHEMAS })).toHaveAttribute(
+    "href",
+    urls.schemas.index,
+  );
+  expect(screen.getByRole("link", { name: "Groups" })).toHaveAttribute(
```

Comment:
question: why not adding `"Groups"` and `"Roles"` to the labels as well?

Reply:
The link labels are defined inside the external ReBAC Admin package so these strings are only used in a single place in this codebase (these tests), but I can add them to the enum as well if you think that would be better?